keyboards/runner3680/4x7/config.h

```diff
@@ -51,9 +51,10 @@
 /* ws2812 RGB LED */
 #define RGB_DI_PIN D3
 #define RGBLIGHT_ANIMATIONS
-#define RGBLED_NUM 70
+#define RGBLED_NUM 28
```

Comment:
Is this accurate? Most other variants have this set to `MATRIX_ROWS * MATRIX_COLS`.

Seems like this file should have

```
#define RGBLED_NUM 56
...
#define RGBLED_SPLIT { 28, 28 }
```

Reply:
Thank you. You are right.